docker-compose-dev.yml

```diff
@@ -2,7 +2,7 @@ version: '3.1'
 services:
 
   redis:
-    image: 'bitnami/redis:5.0'
+    image: 'redis:5.0'
```

Comment:
The bitnami distribution does not build containers with ARM support. Recommended to use redis from the official source. This is only used for running locally anyway